test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -148,42 +151,61 @@ func schedulePods(config *testConfig) int32 {
 			break
 		}
 	}
+
+	scheduled := int32(0)
+	ctx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
use context seems to be overkill here, as we don't either derive a context from the caller or pass the `ctx` generated here to callee.

Prefer to use a simple `chan`.

Reply:
There are two goroutines (main goroutine and the goroutine with ticker) which need to receive the signal. If we use a simple channel here, [only one of them (main goroutine) would receive the value](https://play.golang.org/p/utwSsQ33orj).

Reply:
I see the point. SGTM.